client/src/components/EventView/index.js

```diff
@@ -28,18 +50,26 @@ const EventView = () => {
   return (
     <div>
```

Comment:
I've included the example from the issue description down below. There is a `<Stack>` component which automatically spaces each child element. You can use that instead of the ExtraPadding. 
```
return (
    <div>
      <PageAccess />
      <AppBar color={theme1} />
      {apiStatus.loading ? (
        <CenteredLoadingSpinner
          size={150}
          color={theme1}
          loading={apiStatus.loading}
        />
      ) : (
        <>
          {apiStatus.error ? (
            <Layout>
              <Error fontSize={fontSize24}>
                {apiStatus.message ||
                  "An error occurred. Please try again later."}
              </Error>
            </Layout>
          ) : (
            <>
              < YOUR NEW HEADER HERE />
              <Layout>
                <Stack gapSize={spacing32}>
                <RSVP />
                 <Description />
                 <Location />
                 <Invitees />
                 <InvitedPrompt/>
                </Stack>
              </Layout>
            </>
          )}
        </>
      )}
    </div>
  );
```